packages/aws-cdk-lib/region-info/build-tools/generate-static-data.ts

```diff
@@ -169,10 +169,24 @@ function checkRegionsSubMap(map: Record<string, Record<string, Record<string, un
   }
 }
 
-export function after(region: string, ruleOrRegion: string | symbol) {
+function after(region: string, ruleOrRegion: string | symbol) {
   return region !== ruleOrRegion && !before(region, ruleOrRegion);
 }
 
+/**
+ * Whether or not a region predates a given rule (or region).
+ *
+ * Unknown region => we have to assume no.
+ */
+function before(region: string, ruleOrRegion: string | symbol) {
+  const ruleIx = AWS_REGIONS_AND_RULES.indexOf(ruleOrRegion);
+  if (ruleIx === -1) {
+    throw new Error(`Unknown rule: ${String(ruleOrRegion)}`);
```

Comment:
shouldn't this be Unscoped?

Reply:
`build-tools` are not used at runtime, so excluded. And that's the whole reason why I had to move the function in the first place: it's only used at build time and adding the core module into the hot path completely blows the build script up 💥

I'll update the PR description tomorrow.